site/src/components/timeline.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Timeline
+description: Common timeline component in the Atlas Design System
+template: standard
+classType: Component
+classPrefixes:
+  - timeline
+---
+
+# Timeline
+
+The timeline component is designed to present a chronological sequence of events in a vertical list format. Each item in the timeline represents a specific event. The component is structured to enhance readability and user engagement by providing clear visual markers for each event.
+
+## Timeline badge
+
+The `timeline-item-badge` is recommended to used with the badge component. For component documentation [visit the badge component](../components/badge.md)
+
+
+```html
+<ol class="timeline">
+	<li class="timeline-item">
```

Comment:
I think both to provide a good example and better testing, we need three list items in his example.